delivery_carrier_label_batch/wizard/generate_labels.py

```diff
@@ -0,0 +1,245 @@
+# -*- coding: utf-8 -*-
+# Copyright 2013-2016 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+import Queue
+import logging
+import openerp
+import threading
+from contextlib import closing, contextmanager
```

Comment:
```suggestion
from contextlib import contextmanager
```
`closing` must be removed to make travis happy